projects/canopy/src/lib/carousel/auto-play/auto-play.component.html

```diff
@@ -0,0 +1,20 @@
+<div aria-live="polite" aria-atomic="true" class="lg-carousel-autoplay">
```

Comment:
One thing we tend to to with component wrappers is apply them directly to the host element, using `HostBinding`.

E.g.

```
@HostBinding('class.lg-carousel-autoplay') class = true;
@HostBinding('attr.aria-live') role = 'polite';
@HostBinding('attr.aria-atomic') role = 'true';
```